src/snowflake/snowpark/_internal/data_source/drivers/base_driver.py

```diff
@@ -46,6 +47,49 @@ def infer_schema_from_description(self, table_or_query: str) -> StructType:
             cursor.close()
             conn.close()
 
+    def udtf_ingestion(
+        self,
+        session: "snowflake.snowpark.Session",
+        schema: StructType,
+        partition_table: str,
+        external_access_integrations: str,
+        fetch_size: int = 1000,
+    ) -> "snowflake.snowpark.DataFrame":
+        session.udtf.register(
+            self.udtf_class_builder(fetch_size),
+            name="dbapi",
+            output_schema=StructType(
+                [
+                    StructField(field.name, StringType(), field.nullable)
+                    for field in schema.fields
+                ]
+            ),
+            external_access_integrations=[external_access_integrations],
+            packages=get_dbms_packages(self.create_connection()),
+        )
+        call_udtf_sql = f"""
+            select * from {partition_table}, table(dbapi(partition))
```

Comment:
is the `partition` the single column of the `partition_table`?
I feel we should make a constant to make it more obvious and easy to maintain in the future

Reply:
yes, i made it a constant now